pkg/pipelineascode/pipelineascode.go

```diff
@@ -72,6 +76,47 @@ func (p *PacRun) Run(ctx context.Context) error {
 	return nil
 }
 
+// emitMessage create a kubernetes event on the target namespace with the given
+// message and log it according to the log level
+func (p *PacRun) emitMessage(message, loggerLevel string) {
+	var namespace, name string
+	if p.repoMatched == nil {
```

Comment:
Shouldn't it be `p.repoMatched != nil `? My understanding is repoMatched is a Repository object and so get the values of those parameters when the Repo object is indeed found!

Reply:
yes you right, i really should add proper unit test, thanks..